styles.css

```diff
@@ -0,0 +1,370 @@
+* {
+    margin: 0px;
+    font-family: 'Franklin Gothic Medium', 'Arial Narrow', Arial, sans-serif; 
+}
+
+
+.container {
+    display: grid;
+    grid-template-columns: 1fr 2fr 1.4fr;
+    grid-template-rows: 100vh;
+    margin-left: 15%;
+    margin-right: 15%;
+}
+
+
+/* LEFT MENU */
+.left-menu {
+    display: grid;
+    grid-template-rows:  repeat(9, 40px) 60px 1fr ;
+    grid-template-columns: 50px 1fr;
+    row-gap: 20px;
+
+    width: 250px;
+    margin-left: 20px;
+    padding-top: 10px;
+    align-items: end;
+}
+
+.twitter-logo {
+    grid-column: 1 / span 2;
+}
+
+.nav {
+    text-decoration: none;
+    color: black;
+}
+
+.nav:hover {
+    color: rgb(75, 160, 235);
+}
+
+.nav svg:hover {
+    cursor: pointer;
+}
+
+.tweet_button {
+    background-color: rgb(75, 160, 235);
+    border: none;
+    border-radius: 50px;
+    height: 50px;
+    width: 230px;
+    color: white;
+    font-weight: bold;
+    grid-column: 1 / span 2;
+}
+
+.tweet_button:hover{
+    background-color: rgb(49, 149, 236);
+    cursor: pointer;
+}
+
+.profile-left-menu {
+    display: grid;
+    grid-template-columns: 1fr 2fr 1fr;
+}
+
+.profile-photo {
+object-fit: cover;
+border: solid rgba(0, 0, 0, 0.117);
```

Comment:
Is the a value responsible for transparency?

Reply:
Yes. This leaves a very light grey transparent halo around the profile pic. Which will then change when hovered on.